modules/lws-install-operator.adoc

```diff
@@ -0,0 +1,46 @@
+// Module included in the following assemblies:
+//
+// * ai_workloads/leader_worker_set/lws-managing.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="lws-install-operator_{context}"]
+= Installing the {lws-operator}
+
+You can use the web console to install the {lws-operator}.
+
+////
+Questions/TODO:
+- Will the update channel stay stable-v1.0, or will it be just 'stable'?
```

Comment:
@ardaguclu I've seen most other operators use just `stable` as their update channel. Just checking that this one is staying as `stable-v1.0` for GA?

Reply:
This is auto-generated by the tool we used to generate Catalog files. I can also see that kueue https://github.com/openshift/kueue-fbc/blob/fc3abb0587c0b3bbeb71a5914c2df18b311e77eb/v4.19/catalog/kueue-operator/catalog.json#L4 uses this channel. 

I believe that we are fine.